### ISSUE TITLE
np.int deprecated APIs: Windows current release is broken; please consider doing a release (fix already merged)

### ISSUE THREAD

**Issue by gradientsky** [fastai/fastai#3870]
Be sure you've searched [the forums](https://forums.fast.ai) for the error message you received. Also, unless you're an experienced fastai developer, first ask on the forums to see if someone else has seen a similar issue already and knows how to solve it. **Only file a bug report here when you're quite confident it's not an issue with your local setup.**

**Please see [this model example](https://github.com/fastai/fastai2/issues/487) of how to fill out an issue correctly.  Please try to emulate that example as appropriate when opening an issue.**

Please confirm you have the latest versions of fastai, fastcore, and nbdev prior to reporting a bug (delete one): YES

**Describe the bug**
**TL;DR:** please consider doing a new release to bring [fix done in December](https://github.com/fastai/fastai/commit/6c26a69705675ed8e0a55bbca80635500787d225) live. Windows platform fails without it.

There was a [fix](https://github.com/fastai/fastai/commit/6c26a69705675ed8e0a55bbca80635500787d225) applied to [this line](https://github.com/fastai/fastai/blob/master/fastai/torch_core.py#L136) in December to address the future issues related `np.int` usage; however, Windows version is already failing with the exceptions that API was removed: 
Versions: 
- fastai-2.7.10-py3-none-any.whl
- numpy-1.24.2-cp39-cp39-win_amd64.whl
- fastcore-1.5.28-py3-none-any.whl

**To Reproduce**
1. This test is currently broken on windows: [link](https://github.com/autogluon/autogluon/blob/master/tabular/tests/unittests/models/test_tabular_nn_fastai.py)

**Expected behavior**

**Error with full stack trace**

Place between these lines with triple backticks:

[Full trace](https://github.com/autogluon/autogluon/actions/runs/4121969984/jobs/7118200983#step:6:1458)
```
Fitting model: NeuralNetFastAI ...
	Warning: Exception caused NeuralNetFastAI to fail during training... Skipping this model.
		module 'numpy' has no attribute 'int'.
`np.int` was a deprecated alias for the builtin `int`. To avoid this error in existing code, use `int` by itself. Doing this will not modify any behavior and is safe. When replacing `np.int`, you may wish to use e.g. `np.int64` or `np.int32` to specify the precision. If you wish to review your current use, check the release note link for additional information.
The aliases was originally deprecated in NumPy 1.20; for more details and guidance see the original release note at:
    https://numpy.org/devdocs/release/1.20.0-notes.html#deprecations
Detailed Traceback:
Traceback (most recent call last):
  File "D:\a\autogluon\autogluon\core\src\autogluon\core\trainer\abstract_trainer.py", line 1502, in _train_and_save
    model = self._train_single(X, y, model, X_val, y_val, total_resources=total_resources, **model_fit_kwargs)
  File "D:\a\autogluon\autogluon\core\src\autogluon\core\trainer\abstract_trainer.py", line 1447, in _train_single
    model = model.fit(X=X, y=y, X_val=X_val, y_val=y_val, total_resources=total_resources, **model_fit_kwargs)
  File "D:\a\autogluon\autogluon\core\src\autogluon\core\models\abstract\abstract_model.py", line 703, in fit
    out = self._fit(**kwargs)
  File "D:\a\autogluon\autogluon\tabular\src\autogluon\tabular\models\fastainn\tabular_nn_fastai.py", line 288, in _fit
    self.model.fit_one_cycle(epochs, params['lr'], cbs=callbacks)
  File "C:\hostedtoolcache\windows\Python\3.9.13\x64\lib\site-packages\fastai\callback\schedule.py", line 117, in fit_one_cycle
    scheds = {'lr': combined_cos(pct_start, lr_max/div, lr_max, lr_max/div_final),
  File "C:\hostedtoolcache\windows\Python\3.9.13\x64\lib\site-packages\fastai\callback\schedule.py", line 83, in combined_cos
    return combine_scheds([pct,1-pct], [SchedCos(start, middle), SchedCos(middle, end)])
  File "C:\hostedtoolcache\windows\Python\3.9.13\x64\lib\site-packages\fastai\callback\schedule.py", line 70, in combine_scheds
    pcts = tensor([0] + L(pcts))
  File "C:\hostedtoolcache\windows\Python\3.9.13\x64\lib\site-packages\fastai\torch_core.py", line 154, in tensor
    else _array2tensor(array(x), **kwargs))
  File "C:\hostedtoolcache\windows\Python\3.9.13\x64\lib\site-packages\fastai\torch_core.py", line 136, in _array2tensor
    if sys.platform == "win32" and x.dtype==np.int: x = x.astype(np.int64)
  File "C:\hostedtoolcache\windows\Python\3.9.13\x64\lib\site-packages\numpy\__init__.py", line 305, in __getattr__
    raise AttributeError(__former_attrs__[attr])
AttributeError: module 'numpy' has no attribute 'int'.
`np.int` was a deprecated alias for the builtin `int`. To avoid this error in existing code, use `int` by itself. Doing this will not modify any behavior and is safe. When replacing `np.int`, you may wish to use e.g. `np.int64` or `np.int32` to specify the precision. If you wish to review your current use, check the release note link for additional information.
The aliases was originally deprecated in NumPy 1.20; for more details and guidance see the original release note at:
    https://numpy.org/devdocs/release/1.20.0-notes.html#deprecations
```

**Additional context**

**Comment by warner-benjamin** [comment]
@gradientsky fastai 2.7.11 should resolve this issue

**Comment by gradientsky** [comment]
@warner-benjamin can confirm. The problem is [fixed](https://github.com/autogluon/autogluon/actions/runs/4226644115) - closing.
